packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -281,66 +307,69 @@ class Radio extends React.Component<Props> {
-                // fine for now.
-                // TODO(emily): Come up with a more comprehensive way to solve
-                // this sort of "serialized state breaks when internal
-                // structure changes" problem.
-                correct:
-                    typeof choice.correct === "undefined"
+        const choicesProp: $ReadOnlyArray<ChoiceType> = choices.map(
+            (choice: PerseusRadioChoice, i: number) => {
+                const content =
+                    choice.isNoneOfTheAbove && !choice.content
+                        ? // we use i18n._ instead of $_ here because the content
+                          // sent to a renderer needs to be a string, not a react
+                          // node (/renderable/fragment).
+                          i18n._("None of the above")
+                        : choice.content;
+
+                const {
+                    selected,
+                    crossedOut,
+                    rationaleShown,
+                    correctnessShown,
+                    readOnly,
+                    highlighted,
+                    previouslyAnswered,
+                } = choiceStates[i];
+
+                const reviewChoice =
+                    this.props.reviewModeRubric &&
+                    this.props.reviewModeRubric.choices[i];
+
+                return {
+                    content: this._renderRenderer(content),
+                    checked: selected,
+                    // Current versions of the radio widget always pass in the
+                    // "correct" value through the choices. Old serialized state
+                    // for radio widgets doesn't have this though, so we have to
+                    // pull the correctness out of the review mode rubric. This
+                    // only works because all of the places we use
+                    // `restoreSerializedState()` also turn on reviewMode, but is
+                    // fine for now.
+                    // TODO(emily): Come up with a more comprehensive way to solve
+                    // this sort of "serialized state breaks when internal
+                    // structure changes" problem.
+                    correct: !!(choice.correct === undefined
                         ? !!reviewChoice && reviewChoice.correct
-                        : choice.correct,
-                disabled: readOnly,
-                hasRationale: !!choice.clue,
-                rationale: this._renderRenderer(choice.clue),
-                showRationale: rationaleShown,
-                showCorrectness: correctnessShown,
-                isNoneOfTheAbove: choice.isNoneOfTheAbove,
-                revealNoneOfTheAbove: this.props.questionCompleted && selected,
-                crossedOut,
-                highlighted,
-                previouslyAnswered: previouslyAnswered,
-            };
-        });
+                        : choice.correct),
```

Comment:
**style nit (non-blocking):** I think coercing the "if" result and the "else" blocks to be booleans separately might be a bit easier to read, e.g.
```
correct:
    choice.correct === undefined
        ? !!reviewChoice && !!reviewChoice.correct
        : choice.correct,
```

Reply:
Updated! Thanks~ :D